CHANGELOG.ios.md

```diff
@@ -4,6 +4,17 @@
 - [iOS 2.0 SDK](http://aka.ms/gc6fex)
 - [iOS 1.2.4 SDK](http://aka.ms/kymw2g)
 
+### Version 2.2.0
+- Make ID property of records case-insensitive
+- Return the underlying NSOperation on performing push, pull and purge operations
+- Added support for configuring the page size of a pull operation
+- Added support for NSUUID in PredicateTranslator
+- Added support for turning off data store handling of table operations
+- Fixed bug which caused reads from data store to swallow properties set to nil
+- Fixed handling of network errors during push operations
+- Fixed potential race conditions while performing table operations
+- Fixed incorrect ID validation during insert operations
```

Comment:
in our team oneenote, we have a command that gets the commits that includes the link to the commit.  We used that for managed last time, it would be nice to get those here so each change can be linked to the main commit that made it.  Also the issue if one existed.